tests/utils/test_io.py

```diff
@@ -114,12 +118,15 @@ def test_fingerprint_containers(container):
     ) == rasa.shared.utils.io.deep_container_fingerprint(copy.deepcopy(container))
 
 
-def test_fingerprint_does_not_use_string_hashing(monkeypatch):
-    dictionary = {"a": ["b"], "c": {"d": "e"}}
-    f1 = rasa.shared.utils.io.deep_container_fingerprint(dictionary)
+def test_fingerprint_is_consistent_across_runs(tmp_path):
+    fingerprint_script = """
+        import rasa.shared.utils.io
+        dictionary = {"a": ["b"], "c": {"d": "e"}}
+        print(rasa.shared.utils.io.deep_container_fingerprint(dictionary))"""
+    tests.utilities.fingerprint_consistency_test(fingerprint_script, tmp_path)
+
 
-    # in case we would rely on string hashes anywhere, using a different seed
-    # would lead to different fingerprints and let this test fail
-    monkeypatch.setenv("PYTHONHASHSEED", "42")
```

Comment:
dat never worked 😅 ~ or at least not anymore

Reply:
https://stackoverflow.com/questions/30585108/disable-hash-randomization-from-within-python-program

Reply:
haha good catch 🤦